Parkinsons-Mouse-tutorial.ipynb

```diff
@@ -0,0 +1,3062 @@
+       "                        'static notebook viewer services like nbviewer.)';\n",
+       "    } else if (res.status === 409) {\n",
+       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
+       "                        ' to see the visualization.';\n",
+       "    } else if (res.ok) {\n",
+       "        url = res.url;\n",
+       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
+       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
+       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
+       "                        ' target=\\'_blank\\'>new window</a>'\n",
+       "    } else {\n",
+       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
+       "                        ' errors.';\n",
+       "    }\n",
+       "});\n",
+       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-v2oetses')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
+      ],
+      "text/plain": [
+       "<visualization: Visualization uuid: 43b6ca07-6850-45d7-a0b4-545933b1222d>"
+      ]
+     },
+     "execution_count": 99,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sample_classifier_results.accuracy_results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This pipeline generates a number of output artifacts and visualizations. You can read more about these in the [sample classifier](https://docs.qiime2.org/2020.6/tutorials/sample-classifier/) tutorial but right now let’s just focus on `sample_classifier_results` This visualization tells you how well your sample classifier performed via a [confusion matrix](https://en.wikipedia.org/wiki/Confusion_matrix) and accompanying table of accuracy scores. This tells you how frequently each sample type is classified to each sample class, including the correct class label. Overall error rates are also reported in the table below."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Question\n",
+    "\n",
+    "How did we do? Just for fun, try predicting some of the other metadata columns to see how easily `cage_id` and other columns can be predicted."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Looks like we did pretty well! So we can see what features are most predictive of each sample class (donor and genotype groups). The importance scores are stored in the ./sample-classifier-results/feature_importance.qza artifact (pro tip: this can be view with the qiime metadata tabulate command we covered earlier). Here we will generate a heatmap showing the mean abundance of the 100 most important ASVs in each genotype and donor group."
```

Comment:
the qiime metadata tabulate commannd - powinno byc zapisane w notacji wersji API